picard/ui/theme_detect.py

```diff
@@ -119,13 +120,50 @@ def detect_freedesktop_color_scheme_dark() -> bool:
         log.debug("gsettings get org.freedesktop.appearance.color-scheme failed.")
     return False
 
+def get_current_desktop_environment() -> str:
+    """Detect the current desktop environment (DE) as a lowercase string."""
+    de = os.environ.get("XDG_CURRENT_DESKTOP")
+    if de:
+        return de.lower()
+    # Fallbacks for KDE, XFCE, LXQt
+    if os.environ.get("KDE_FULL_SESSION") == "true":
+        return "kde"
+    if os.environ.get("XDG_SESSION_DESKTOP"):
+        return os.environ["XDG_SESSION_DESKTOP"].lower()
+    if os.environ.get("DESKTOP_SESSION"):
+        return os.environ["DESKTOP_SESSION"].lower()
+    return ""
```

Comment:
Just as a side note: We have similar (but less complete) checks also in `icontheme.py`. We could unify this later (by using `get_current_desktop_environment` also in icontheme). But I would keep this out of this PR.